services/web/server/src/simcore_service_webserver/functions/_controller/_functions_rest.py

```diff
@@ -370,12 +371,18 @@ async def update_function(request: web.Request) -> web.Response:
 async def delete_function(request: web.Request) -> web.Response:
     path_params = parse_request_path_parameters_as(FunctionPathParams, request)
     function_id = path_params.function_id
+
+    query_params: FunctionDeleteQueryParams = parse_request_query_parameters_as(
```

Comment:
Do we have a use-case in which we want to use 'force'? If not, i suggest to remove it from the api and call the service-layer function with `forge=True`